Sources/Sentry/Public/SentrySession.h

```diff
@@ -11,7 +12,7 @@ typedef NS_ENUM(NSUInteger, SentrySessionStatus) {
     kSentrySessionStatusAbnormal = 3,
 };
 
-@interface SentrySession : NSObject <NSCopying>
+@interface SentrySession : NSObject <SentrySerializable, NSCopying>
```

Comment:
Just curious, why do we need this protocol if it was compiling/running without it?

Reply:
That's a good point. I will check that.